numpy/core/tests/test_fromnumeric.py

```diff
@@ -1,17 +0,0 @@
-from __future__ import division, absolute_import, print_function
```

Comment:
Hmm, at some point we need to reorganize `numpy/core`, but I suppose that is for another day.